package.json

```diff
@@ -38,7 +38,7 @@
     "test:coverage": "jest test --coverage",
     "test:docs": "npm run docs && npm run verify:docs",
     "test:only": "jest test",
-    "verify:docs": "git diff --exit-code ./src/README.md || echo Run `npm run docs` and commit updated `src/README.md` file."
+    "verify:docs": "sh ci/verify-docs.sh"
```

Comment:
Why not just update the docs instead of failing this?

Reply:
It would need to create a commit behind the scenes which would be "too automated" I think.
I think that user attention is necessary as manual check of what documentation.js generates is also necessary.

Reply:
maybe just put it in stagging is good enough

Reply:
Hmm... I'm not sure it's necessary her. Npm task name is 'verify:docs', so it should not do much more than verify if it is up to date, imho. There's another script - "test:docs" - which creates updates 'src/README.md' file.

Reply:
I would think to save the next step for the user, but it's not critical

Reply:
Still, I'm not convinced about that. It's something that you need to check before commiting and by changing configuration in `documentation.yml` you can adjust it. So you would need to stage it few times. I really would leave it as it is.